<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dbexport</class>
 <widget class="QWidget" name="dbexport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DB-Excel</string>
  </property>
  <widget class="QLineEdit" name="le_db">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_OpenDB">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览db</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Export">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>导出</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_excel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>数据库：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Excel：</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>341</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="Optwidget" native="true">
   <property name="geometry">
    <rect>
     <x>279</x>
     <y>199</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
